MetaMask/metamask-extension
# Snap removal confirmation

Fixes: #13414

Explanation:  this PR adds a new `SnapRemoveWarning` component to display a user confirmation on Snap removal.

**Final look:**

<img width="359" alt="image" src="https://user-images.githubusercontent.com/38461171/157066462-321f6e7d-d37e-43cb-8b87-619f92a408ea.png">
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/38461171/157066621-9f49fcd2-d4e6-4015-ac4c-4ec357af311d.png">

Manual testing steps:  

- Run yarn start --build-type flask
- To test a snap with the key access permissions, you can clone our fork of the filecoin snap at: [Filecoin snap](https://github.com/MetaMask/filsnap/tree/prerelease)
- Checkout the prerelease branch and then run yarn && yarn demo
- Follow the install flow for the filecoin snap from the locally hosted dapp and observe the changes when the snap install warning pops up
